server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/json/Serializer.scala

```diff
@@ -103,17 +102,31 @@ class Serializer {
 
   private implicit val capabilitiesWrites: Writes[Capabilities] = capabilities => setCapabilityWrites.writes(Set(capabilities.coreCapability, capabilities.mailCapability))
 
-  private implicit def identifierMapWrite[Any](implicit idWriter: Writes[Id]): Writes[Map[CapabilityIdentifier, Any]] =
+  private implicit val accountIdWrites: Format[AccountId] = Json.valueFormat[AccountId]
+  private implicit def identifierMapWrite[Any](implicit idWriter: Writes[AccountId]): Writes[Map[CapabilityIdentifier, Any]] =
     (m: Map[CapabilityIdentifier, Any]) => {
       m.foldLeft(JsObject.empty)((jsObject, kv) => {
-        val (identifier: CapabilityIdentifier, id: Id) = kv
+        val (identifier: CapabilityIdentifier, id: AccountId) = kv
         jsObject.+(identifier.value, idWriter.writes(id))
       })
     }
 
   private implicit val isPersonalFormat: Format[IsPersonal] = Json.valueFormat[IsPersonal]
   private implicit val isReadOnlyFormat: Format[IsReadOnly] = Json.valueFormat[IsReadOnly]
-  private implicit val accountWrites: Writes[Account] = Json.writes[Account]
+  private implicit def accountWrites(implicit usernameWrites: Writes[Username],
+                                     isPersonalWrites: Writes[IsPersonal],
+                                     isReadOnlyWrites: Writes[IsReadOnly],
+                                     setWrites: Writes[Set[_ <: Capability]]): Writes[Account] =
+    (account: Account) => JsObject(Seq(
+      ("name", usernameWrites.writes(account.name)),
```

Comment:
this is not sophisticated. I tried to follow the sample using `(JsPath \ ...` above but it doesn't work

Reply:
by some helps from Remi plus my wisdom, I had resolved it